stylua-vscode/package.json

```diff
@@ -42,6 +42,27 @@
     "configuration": {
       "title": "StyLua",
       "properties": {
+        "stylua.releaseVersion": {
+          "type": "string",
+          "default": "latest",
+          "enum": [
+            "latest",
+            "v0.9",
+            "v0.8",
+            "v0.7",
+            "v0.6",
+            "v0.5"
+          ],
+          "enumDescriptions": [
+            "The most recent version released. This will always keep you up to date."
```

Comment:
I know its already mentioned in `stylua.targetReleaseVersion`, but is it also worth saying something along the lines of "This setting is overriden by #stylua.targetReleaseVersion# if provided", for completeness sake?

Reply:
I looked at other extensions and chose this way because others don't seem to reflect settings this way. It makes sense that a user normally using `releaseVersion` would not generally care about `targetReleaseVersion` while setting it, but the reverse is always true. That's just my rationale, I'll leave this final decision up to you. Let me know and I'll make the change.

Reply:
Its an incredibly minor point, so doesn't really matter if we don't add it, but I think it helps in the unlikely event that a user forgets that they have a targetReleaseVersion set - it just points them to the setting in case they are wondering why it isn't taking affect.